### PR TITLE
Rewrite triggers.md for clarity and flow

### PR DESCRIPTION
**Pull request by sergetron**
# Changes

Rewrites triggers.md for clarity and flow

# Submitter Checklist

These are the criteria that every PR should meet, please check them off as you
review them:

- [ ] Includes [tests](https://github.com/tektoncd/community/blob/master/standards.md#tests) (if functionality changed/added)
- [X] Includes [docs](https://github.com/tektoncd/community/blob/master/standards.md#docs) (if user facing)
- [ ] Commit messages follow [commit message best practices](https://github.com/tektoncd/community/blob/master/standards.md#commits)
- [ ] Release notes block has been filled in or deleted (only if no user facing changes)

_See [the contribution guide](https://github.com/tektoncd/triggers/blob/master/CONTRIBUTING.md) for more details._

# Release Notes

```release-note
NONE
```

### CODE REVIEW

File: docs/triggers.md [hunk 1]
```diff
@@ -4,33 +4,26 @@ linkTitle: "Trigger"
...
-    object.
-  - [`metadata`][kubernetes-overview] - Specifies data to uniquely identify the
-    `Trigger` resource object, for example a `name`.
-  - [`spec`][kubernetes-overview] - Specifies the configuration information for
-    your Trigger resource object. The spec include:
+  - [`apiVersion`][kubernetes-overview] - Specifies the API version; for example `triggers.tekton.dev/v1alpha1`.
+  - [`kind`][kubernetes-overview] - Specifies that this resource object is a `Trigger` object.
+  - [`metadata`][kubernetes-overview] - Specifies metadata to uniquely identify this `Trigger` object; for example a `name`.
+  - [`spec`][kubernetes-overview] - Specifies the configuration information for this Trigger object, including:
     - [`bindings`] - (Optional) A list of bindings to use. Can either be a reference to existing `TriggerBinding` resources or embedded name/value pairs.
```
Comment (dibyom) [inline docs/triggers.md:21]: We should update the bindings description to match the others too: .Specfies a list of bindings to use etc.

Comment (sergetron) [reply docs/triggers.md]: Fixed!

**Review by dibyom — APPROVED** [review]
/lgtm

**Pull request merged**